spec/request/api/v1/targets/destroy_target_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe " DELETE /api/v1/targets", :type => :request do  
+  let(:user) { create(:user) }  
+  let!(:target) { create(:target, user: user) }
+
+  context 'when the request is not valid' do
+    it 'returns unauthorized response' do        
+      delete api_v1_target_path(id: target.id),  as: :json
+      expect(response).to have_http_status(:unauthorized)
+    end
+  end
+
+  context 'when the request is valid' do
+    subject do
+      delete api_v1_target_path(id: target.id), params:target ,headers: user.create_new_auth_token, as: :json
+    end 
+
+    it 'returns a saccessful response' do     
+      subject
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'delete a target correctly' do     
+      expect{subject}.to change{Target.count}.by(-1)
```

Comment:
Leave spaces in between the curly braces, this is something that rubocop should tell you once you have it on the project